health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -39,7 +39,7 @@ const UserDownload = () => {
         }
     }, [data]); // Only run this effect when `data` changes
 
-    
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)